config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -1,3 +1,56 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kind-conformance-dual
+    optional: true
+    always_run: false
+    skip_report: false
+    decorate: true
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decoration_config:
+      timeout: 60m
+      grace_period: 15m
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/krte:v20200720-4766100-master
+        command:
+        - wrapper.sh
+        - bash
+        - -c
+        - curl -sSL https://github.com/aojea/kind/releases/download/dualstack/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
```

Comment:
this was to live in my fork until it is more stable
cc: @BenTheElder

Reply:
ACK, we do have a bit of a chicken and egg problem here, I suppose this is reasonable in the immediate term.
We've had similar things in CI before.

Reply:
we are blind against a PR with 73 files changes and +9,512 −3,250 :shrug:

Reply:
long term we don't keep things like this, but notably early windows support required a similar fork for kube-up, temporarily.

Reply:
this is temporary, of course